config/config.go

```diff
@@ -1,35 +1,66 @@
 package config
 
 import (
-	"fmt"
 	"log"
-	"path/filepath"
-
-	"github.com/spf13/viper"
+	"os"
 )
 
-// Init is an exported method that takes the environment starts the viper
-// (external lib) and returns the configuration struct.
-func Init(cfgFile string) {
-	if cfgFile != "" { // enable ability to specify config file via flag
-		viper.SetConfigFile(cfgFile)
-	} else {
-		viper.SetConfigName(".taco") // name of config file (without extension)
-		viper.AddConfigPath("$HOME") // adding home directory as first search path
+type Config struct {
+	AWS_Dynamo_Region string
+	Dynamo_Db         string
+	Disable_SSL       bool
+	Table_Name        string
+}
+
+func NewConfig() *Config {
+	return &Config{
+		AWS_Dynamo_Region: aws_dynamo_region(),
+		Dynamo_Db:         dynamo_db(),
+		Disable_SSL:       disable_ssl(),
+		Table_Name:        table_name(),
+	}
+}
+
+func aws_dynamo_region() string {
+	var region string
+	region = os.Getenv("AWS_DYNAMO_REGION")
```

Comment:
The taco API should always live in the same region as the dynamo end point, so the env var of `AWS_REGION` is sufficient. Same goes for the S3 bucket and the kinesis endpont